docs/planning/backend/app/services/fragella_client.py

```
@@ -0,0 +1,341 @@
+        """Load usage tracking from file."""
+        self.usage = {"month": None, "count": 0, "requests": []}
+
+        if self.usage_file.exists():
+            try:
+                with open(self.usage_file) as f:
+                    self.usage = json.load(f)
+            except (json.JSONDecodeError, IOError):
+                pass
+
+        # Reset if new month
+        current_month = date.today().strftime("%Y-%m")
```

Ruff (0.14.10)

`datetime.date.today()` used

(DTZ011)